packages/design-system/src/components/TextField/Mask.tsx

```diff
@@ -36,6 +36,10 @@ export class Mask extends React.PureComponent<MaskProps, any> {
     const field = this.field();
     const initialValue = field.props.value || field.props.defaultValue;
 
+    console.warn(
+      '[Deprecated]: Please utilize the LabelMask prop in place of the Mask prop. Mask will be deprecated for use in the next major release'
```

Comment:
Should we have a ticket in our backlog to track this?

Reply:
We can't quite deprecate the old one. I think we should do this in a separate PR under a separate ticket. (And for that reason we should also keep the old all-masks story around.)